scripts/build.sh

```diff
@@ -0,0 +1,47 @@
+# Create a browser extension release package
+
+# 1. Capture browser type from argument
+browserType=$1
+
+# 2. Construct release zip filename
+zipName=$browserType-release.zip
+
+# 3. Ensure release directory exists (create if needed)
+mkdir -p build/release/
+
+# 4. Clean up any existing release zip
+rm -f build/release/$browserType-release.zip
+
+# 5. Rename base manifest file
+cd src/ && mv manifest.json manifest.base.json
+if [[ "$browserType" == "firefox" ]]; then
+    faviconUrl='assets/images/favicon.icon'
+    # Remove unnecessary properties for Firefox manifest
+    jq 'del(.background.service_worker, .background.type)' manifest.base.json >manifest.json
```

Comment:
We should do a check at the beginning and throw an error if `jq` is missing, before we do other stuff.

Reply:
that's a good idea